### ISSUE TITLE
physX couldn't use GPU pipeline 

### ISSUE THREAD

**Issue by beastsenpai213** [clearlab-sustech/Wheel-Legged-Gym#8]
Excuse me,
when I launch the training with gpu :
 python wheel_legged_gym/scripts/train.py --task=wheel_legged_vmc_flat --sim_device=cpu --rl_device=cuda
It occur an error on physX that GPU is unavailable,***though the training could keep running on gpu***
Is that an important issue for training? 
```
*** Warning: failed to preload CUDA lib
*** Warning: failed to preload PhysX libs
Importing module 'gym_38' (/home/tianbot/IsaacGym_Preview_4_Package/isaacgym/python/isaacgym/_bindings/linux-x86_64/gym_38.so)
Setting GYM_USD_PLUG_INFO_PATH to /home/tianbot/IsaacGym_Preview_4_Package/isaacgym/python/isaacgym/_bindings/linux-x86_64/usd/plugInfo.json
PyTorch version 2.4.1+cu121
Device count 1
/home/tianbot/IsaacGym_Preview_4_Package/isaacgym/python/isaacgym/_bindings/src/gymtorch
Using /home/tianbot/.cache/torch_extensions/py38_cu121 as PyTorch extensions root...
Emitting ninja build file /home/tianbot/.cache/torch_extensions/py38_cu121/gymtorch/build.ninja...
Building extension module gymtorch...
Allowing ninja to set a default number of workers... (overridable by setting the environment variable MAX_JOBS=N)
ninja: no work to do.
Loading extension module gymtorch...
Can't use GPU pipeline with CPU Physics. Changing pipeline to 'CPU'.
Setting seed: 1
Not connected to PVD
Physics Engine: PhysX
Physics Device: cpu
GPU Pipeline: disabled
